src/React.MSBuild/TransformBabel.cs

```diff
@@ -46,6 +48,9 @@ public override bool Execute()
 			config
 				.SetReuseJavaScriptEngines(false);
 
+			JsEngineSwitcher.Current.DefaultEngineName = MsieJsEngine.EngineName;
+			JsEngineSwitcher.Current.EngineFactories.AddMsie();
```

Comment:
I wonder if a singleton instance of `JsEngineSwitcher` should be registered in the IoC container rather than using the static `Current` property.